src/components/views/rooms/NotificationBadge/UnreadNotificationBadge.tsx

```diff
@@ -23,11 +23,14 @@ import { StatelessNotificationBadge } from "./StatelessNotificationBadge";
 interface Props {
     room?: Room;
     threadId?: string;
-    type?: "badge" | "dot";
+    /**
+     * If true, the badge will always be displayed as a dot.
+     */
```

Comment:
I think this needs updating in much the same way as `StatelessNotificationBadge`'s version (though with differences to account for the fact that `count` comes from a different place)

Reply:
Done in 37e972ea20